style.css

```diff
@@ -112,3 +117,244 @@ ul {
   width: 20px;
   height: 20px;
 }
+
+.project_contents {
+  display: grid;
+  row-gap: 88px;
+  justify-content: center;
+  padding: 114px 24px;
+  list-style-type: none;
+}
+
+.project {
+  padding: 16px 16px 24px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+}
+
+.project img {
+  width: 100%;
+}
+
+.project_description {
+  padding: 16px;
+}
+
+.project_title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  margin-top: -10px;
+  margin-left: -12px;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  text-align: left;
+}
+
+.project_info {
+  display: flex;
+  align-items: center;
+  column-gap: 32px;
+  margin-top: -30px;
+}
+
+.project_title_II {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  margin-left: -8px;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.project_job_title_info_container {
+  display: flex;
+  margin-left: -60px;
+  list-style-type: none;
+}
+
+.job_title_info {
+  font-weight: 600;
+  margin-right: 10px;
+  font-size: 12px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.job_title_info::before {
+  content: url('images/Counter.png');
+  margin-right: 10px;
+}
+
+.project_full_desc {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.project_tags_container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  margin: 16px 0 29px 0;
+}
+
+.project_tag {
+  padding: 4px 12px;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  background: #ebebff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.view_project_btn {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  width: 124px;
+  height: 48px;
+  border-radius: 8px;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  background-color: #fff;
+}
```

Comment:
- [ ] Nice job for making this button look the same as in the Figma design. You can make it look even super by adding the  _Active_, _Hover_, _Pressed_ button styling effects. buttons are a vital element in creating a positive and productive user experience.

Reply:
Yeah, Thanks for noticing this. there is a pressed button effect, why I did not add the hover is because its a mobile version. you can't do hover on mobile.